Location.js

```diff
@@ -0,0 +1,21 @@
+class Location {
+    constructor(locationid, displayName, deviceid, phonenumber, detectionzoneMin, detectionzoneMax, sensitivity, led, noisemap, movThreshold, durationThreshold, stillThreshold, rpmThreshold, xethruSentAlerts, xethruHeartbeatNumber) {
+        this.locationid = locationid,
+        this.displayName = displayName,
+        this.deviceid = deviceid,
+        this.phonenumber = phonenumber,
+        this.detectionzoneMin = detectionzoneMin,
+        this.detectionzoneMax = detectionzoneMax,
+        this.sensitivity = sensitivity,
+        this.led = led,
+        this.noisemap = noisemap,
+        this.movThreshold = movThreshold,
+        this.durationThreshold = durationThreshold,
+        this.stillThreshold = stillThreshold,
+        this.rpmThreshold = rpmThreshold,
+        this.xethruSentAlerts = xethruSentAlerts,
+        this.xethruHeartbeatNumber = xethruHeartbeatNumber
```

Comment:
Does this class purposely not have all the columns from the DB? It looks like it's missing:
* unrespondedTimer
* autoResetThreshold
* TwilioNumber
* DoorStickinessDelay
* UnrespondedSessionTimer
* FallbackPhonenumber

Reply:
I didn't see those in my local db after running migrations so I will double check! We aren't using any of those in the dashboard though (or a bunch of the other ones) so wherever they are being used they are working fine without being part of the object I think, the object is only used for displaying the fields included in the template